src/bezier.cpp

```diff
@@ -7,6 +7,7 @@
 
 #include <unsupported/Eigen/MatrixFunctions>
 
+inline double factorial(uint k) { return std::tgamma(k+1);}
 inline double binomial(uint n, uint k) { return std::tgamma(n + 1) / (std::tgamma(k + 1) * std::tgamma(n - k + 1)); }
```

Comment:
ako je definiran factorial, onda zamijeniti to svuda gdje se tgamma koristi izravno (npr. 3 puta odma u redu ispod)